packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -331,4 +296,24 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
 
     return false;
   }
+
+  private _isGroupExpanded(group: INavLinkGroup): boolean {
+    if (group.name && this.state.isGroupCollapsed.hasOwnProperty(group.name)) {
+      return !this.state.isGroupCollapsed[group.name];
+    }
+    if (group.collapseByDefault !== undefined) {
+      return !group.collapseByDefault;
+    }
+    return true;
+  }
+
+  private _toggleCollapsed(group: INavLinkGroup): void {
+    if (group.name) {
+      const newGroupCollapsed = {
+        ...this.state.isGroupCollapsed, // Make a copy in order to not modify state
+        [group.name]: this._isGroupExpanded(group) // sic - presently open will be collapsed after setState
```

Comment:
ITT this needs to be `!this._isGroupExpanded(group)` otherwise it will never uncollapse.

Reply:
Ah, wait I see now, it's taken care of in _isGroupExpanded.